custom/icds_reports/views.py

```diff
@@ -1709,7 +1717,10 @@ def get(self, request, *args, **kwargs):
         if state_name not in self.valid_state_names:
             return JsonResponse(self.message('invalid_state'), status=400)
 
-        dump = DishaDump(state_name, query_month)
+        dump = DishaDump(state_name, query_month, till_level)
+        if request.user.is_superuser and request.GET.get('rebuild', '') == 'true':
```

Comment:
Sounds a bit scary that someone can hit rebuild constantly using this API param. Hoping that the dangers are thought through and this won't be a huge problem.

Reply:
a superuser?

Reply:
I feel we are overloading the API by adding a way to refresh param here itself